javascript/code-challenges/challenges-11.test.js

```diff
@@ -0,0 +1,230 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1 - Review
+
+Write a function named transformToLis that, given an object, returns an array of the key value pairs as html list items.
+
+For example:
+{
+  name: 'bob',
+  age: 32
+}
+
+Becomes:
+[
+<li>name: bob</li>,
+<li>age: 32</li>
+]
+------------------------------------------------------------------------------------------------ */
+
+function transformToLis(obj){
+  // Solution code here...
+
+  let newArr = [];
+  if(obj.name){
+    let item1 = `<li>name: ${obj.name}</li>`;
+    let item2 = `<li>age: ${obj.age}</li>`;
+    newArr.push(item1, item2);
+  }
+  return newArr;
+};
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 2
+
+Write a function named count that, given an integer and an array of arrays, uses either filter, map, or reduce to count the amount of times the integer is present in the array of arrays.
+
+Note: You might need to use the same method more than once.
+
+For example, count(5, [[1, 3, 5, 7, 9], [5, 5, 5], [1, 2, 3]]) returns 4.
+------------------------------------------------------------------------------------------------ */
+
+const count = (target, input) => {
+  let aoa = 0;
+  for (let array of input){
+    for (let num of array){
+      if (num === target){
+        aoa += 1;
+      }
+    }
+  }
+  return aoa; // Solution code here...
+};
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 3
+
+Write a function that, given an array of integer arrays as input, calculates the total sum of all the elements in the array.
+
+You may want to use filter, map, or reduce for this problem, but are not required to. You may need to use the same method more than once.
+
+For example, [[1, 2, 3, 4, 5], [6, 7, 2, 4, 5, 7], [9, 2, 3, 6,]] returns 66.
+------------------------------------------------------------------------------------------------ */
+
+const totalSum = (input) => {
+  let total = 0;
+  for (let array of input){
+    for (let num of array){
+      total += num;
+    }
+  }
+  return total;// Solution code here...
```

Comment:
Nice work so far on challenges!

Try if you can to work in a higher order function/several for practice :) If you want to come back and do that later, that works fine too. 👍